sql/moz-fx-data-shared-prod/telemetry/mobile_usage_2021/view.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.mobile_usage_2021`
+AS
+WITH base AS (
+  SELECT
+    *,
+    IF(product LIKE '% iOS', 'iOS', 'Android') AS os,
+  FROM
+    `moz-fx-data-shared-prod.telemetry_derived.mobile_usage_v1`
+),
+with_product_info AS (
+  SELECT
+    *,
+    mozdata.analysis.klukas_product_info(product, os).*,
```

Comment:
I will update this once https://github.com/mozilla/bigquery-etl/pull/1812 lands.